actions/appConfig.js

```diff
@@ -55,7 +55,10 @@ class AppConfig {
         this.configMap.certPassword = params.certPassword;
         this.configMap.certKey = params.certKey;
         this.configMap.certThumbprint = params.certThumbprint;
+        this.configMap.enablePreview = this.getJsonFromStr(params.enablePreview, []);
         this.configMap.helixAdminApiKeys = this.getJsonFromStr(params.helixAdminApiKeys);
+        this.configMap.bulkPreviewCheckInterval = parseInt(params.bulkPreviewCheckInterval || '30', 10);
+        this.configMap.maxBulkPreviewChecks = parseInt(params.maxBulkPreviewChecks || '30', 10);
```

Comment:
@arshadparwaiz For the functionality to work with stage AIO API, the github variables need to be added. So do make those additions for these new variables. You should already have access to github secrets.

Reply:
Sure. Thanks @sukamat